src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/posthearingreview/PostHearingReviewSubmittedHandler.java

```diff
@@ -73,14 +73,14 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
 
         caseData = ccdCallbackMapService.handleCcdCallbackMap(callbackMap, caseData);
 
-        handlePostHearingRefused(caseData);
+        handleSetAsideRefused(caseData);
 
         SscsUtil.clearPostHearingFields(caseData);
 
         return new PreSubmitCallbackResponse<>(caseData);
     }
 
-    private void handlePostHearingRefused(SscsCaseData caseData) {
+    private void handleSetAsideRefused(SscsCaseData caseData) {
```

Comment:
Does this need to be renamed? It was created with extension to all post hearing types in mind.

Reply:
Since we can updated the refused states to dormant in CCD then what would it be extended for? I've just refactored it again but kept it with just handling set aside refused in mind

Reply:
Use naming aligned with the scope of method, so it's name is suggestive of what it is doing at the moment of implementing. If in future to be extended, you can eventually add TODO